pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -506,7 +506,7 @@ func newSquashDictHint(dictAccesses, ptrDiff, nAccesses, bigKeys, firstKey hinte
 			//  __squash_dict_max_size is always in scope and has a value of 2**20,
 			squashDictMaxSize := uint64(1048576)
 			if nAccessesValue > squashDictMaxSize {
-				return fmt.Errorf("squash_dict() can only be used with n_accesses<={%d}. Got: n_accesses={%d}.", squashDictMaxSize, nAccessesValue)
+				return fmt.Errorf("squash_dict() can only be used with n_accesses<={%d}. Got: n_accesses={%d}", squashDictMaxSize, nAccessesValue)
```

Comment:
If I remember correctly, I think there were many of these formatting issues with go in the code base. Maybe we can do them in this PR or another PR ?

Reply:
I'm not sure there are still after these one in zero package
If you find some open another one if you want

Reply:
okay.